p4-16/psa/examples/psa-example-resubmit.p4

```diff
@@ -157,15 +139,27 @@ control ingress(inout headers hdr,
     }
 
     apply {
+        // Note that this example will resubmit all input packets
+        // forever, until and unless the PSA implementation drops
+        // them.
         t.apply();
+
+        // Note: For a more complete example, there should be
+        // assignments to meta.resubmit_meta.selector, and the
+        // appropriate one of meta.resubmit_meta.header_a or
+        // meta.resubmit_meta.header_b
     }
 }
 // END:Resubmit_Example_Part2
 
-parser EgressParserImpl(packet_in buffer,
+parser EgressParserImpl(
+    packet_in buffer,
     out headers parsed_hdr,
     inout metadata user_meta,
     in psa_egress_parser_input_metadata_t istd,
+    in metadata normal_meta,
```

Comment:
normal_meta is not used in egress parser, use `empty_metadata_t`?

Reply:
Good catch. I made that change, too.